compiler/modules/front/stmt.bal

```diff
@@ -490,14 +513,16 @@ function validLoopAssignments(StmtContext cx, Assignment[] assignments) returns
 
 function codeGenBreakContinueStmt(StmtContext cx, bir:BasicBlock startBlock, Environment env, s:BreakContinueStmt stmt) returns CodeGenError|StmtEffect {
     bir:Label dest = stmt.breakContinue == "break"? check cx.onBreakLabel(stmt.startPos) : check cx.onContinueLabel(stmt.startPos);
-    bir:BranchInsn branch = { dest, pos: stmt.startPos };
-    startBlock.insns.push(branch);
+    bir:BranchInsn branch;
```

Comment:
Add a local variable `backward` and construct the branch outside the conditional.